cmd/mount.go

```diff
@@ -401,24 +404,70 @@ var mountCmd = &cobra.Command{
 			}
 
 			ctx, _ := context.WithCancel(context.Background()) //nolint
-			daemon.SetSigHandler(sigusrHandler(pipeline, ctx), syscall.SIGUSR1, syscall.SIGUSR2)
+
+			// Signal handlers for parent and child to communicate success or failures in mount
+			var sigusr2, sigchild chan os.Signal
+			if !daemon.WasReborn() { // execute in parent only
+				sigusr2 = make(chan os.Signal, 1)
+				signal.Notify(sigusr2, syscall.SIGUSR2)
+
+				sigchild = make(chan os.Signal, 1)
+				signal.Notify(sigchild, syscall.SIGCHLD)
+			} else { // execute in child only
+				daemon.SetSigHandler(sigusrHandler(pipeline, ctx), syscall.SIGUSR1, syscall.SIGUSR2)
+				go func() {
+					_ = daemon.ServeSignals()
+				}()
+			}
+
 			child, err := dmnCtx.Reborn()
 			if err != nil {
 				log.Err("mount : failed to daemonize application [%v]", err)
 				return Destroy(fmt.Sprintf("failed to daemonize application [%s]", err.Error()))
 			}
 
 			log.Debug("mount: foreground disabled, child = %v", daemon.WasReborn())
-			if child == nil {
+			if child == nil { // execute in child only
 				defer dmnCtx.Release() // nolint
 				setGOConfig()
 				go startDynamicProfiler()
 
 				err = runPipeline(pipeline, ctx)
 				if err != nil {
+					pid := os.Getpid()
+					fname := fmt.Sprintf("/tmp/blobfuse2.%v", pid)
+					if err = ioutil.WriteFile(fname, []byte(err.Error()), 0777); err != nil {
```

Comment:
libfuse print out error message by stderr, you should use [SetLogFile](https://github.com/sevlyar/go-daemon/pull/90/files#diff-bf608eab570a8660c0f9a7c6712ee27da25ecb3246c8aaad0131f7005a8a675aR56-R59) to set stderr as PIPE or [set log file name](https://github.com/sevlyar/go-daemon/blob/master/daemon_unix.go#L24) as this file. However, it can only be done before creating child process.

Reply:
```shell
git checkout 4c9ab71dadf7b4f79cb992655962ed20598d9ca6
```
or 
```shell
git checkout 1c304b820c7c02e0b68fc25cae34071cbef589ca
```
and build blobfuse2 you can see the error message returned by libfuse in terminal.